server/backend/src/users/users.service.ts

```diff
@@ -29,6 +31,42 @@ export class UsersService {
     return user.groups;
   }
 
+  /**
+   * Register a camera to the given user and group
+   * 
+   * @param userId the user's ID.
+   * @param groupId the group ID.
+   * @param name New camera's name
+   * @param pubKey New camera's public key
+   * @param serial New camera's serial
+   */
+  public async putCamera(
+    userId: string,
+    groupId: string,
+    name: string,
+    pubKey: string,
+    serial: string
```

Comment:
ESLint/Prettier really makes this look yucky, can we collect some of the params into a type? Maybe the ones common between the request body and this function?